src/drivers/driver_postgres.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(feature = "driver-postgres")]
+use drivers::Driver as Driv;
+
+pub struct Driver{
+    target: &'static str,
+}
+
+impl Driv for Driver {
```

Comment:
Difference between `Driv` and `Driver` is hard to understand. Maybe `Driver` and `PostgresDriver`?